### ISSUE TITLE
Using an https source for an image doesn't work in the RStudio viewer

### ISSUE THREAD

**Issue by daattali** [rstudio/rstudio#2177]
This is a clone of https://github.com/rstudio/shiny/issues/1148 -- that issue was closed because it's an IDE bug, so I wanted to post here so that I (and others) can track it and see when it gets fixed. I realize it's likely that this issue does exist in your internal bug tracker system, if the duplication is a headache then feel free to close this.

---

RStudio Version : Desktop 1.1.414
OS Version      : Windows 7

Run the following shiny app:

```
library(shiny)

ui <- fluidPage(
  img(src = "https://www.rstudio.com/wp-content/uploads/2014/03/blue-125.png")
)

server <- function(input, output, session) {

}

shinyApp(ui = ui, server = server)
```

Inside the rstudio viewer pane, https images do not show up. In a real browser they do.

**Comment by dfalty** [comment]
Thanks for the report! I can reproduce this with 1.1.423 on Win 8.1.  
![image](https://user-images.githubusercontent.com/15237845/36211560-2f35c85c-1167-11e8-8def-6ac65bf94461.png)

**Comment by jmcphers** [comment]
This should be fixed in RStudio 1.2. Let us know if it isn't!

**Comment by TTricky** [comment]
I thought I'd add my comments since I updated RStudio from stable 1.1.447 to daily 1.2.582 and my related error was fixed (failing to load map images from a https source for the leaflet package). Thank you :)
